Shared/Samples/Add ENC exchange set/README.md

```diff
@@ -30,7 +30,9 @@ Run the sample and view the ENC data. Pan and zoom around the map. Take note of
 
 ## Offline data
 
-This sample downloads the [ENC Exchange Set without updates](https://www.arcgis.com/home/item.html?id=9d2987a825c646468b3ce7512fb76e2d) and [Hydrography dataset resources](https://www.arcgis.com/home/item.html?id=c16f3845b9cc4b93a908d87d28823afd) items from ArcGIS Online.
+This sample downloads the [ENC Exchange Set without updates](https://www.arcgis.com/home/item.html?id=9d2987a825c646468b3ce7512fb76e2d) item from ArcGIS Online.
+
+The latest [hydrography package](https://www.arcgis.com/home/item.html?id=c16f3845b9cc4b93a908d87d28823afd) can be downloaded from ArcGIS for Developers. The S57DataDictionary.xml file is contained in it along with many others but a user does not need to know that in order to render ENC data.
```

Comment:
I don't think we normally deviate from the common design in the READMEs. @yo1995 are we okay to deviate here?

Reply:
This is an intended change. We use to have it in the iOS README: https://github.com/Esri/arcgis-runtime-samples-ios/tree/main/arcgis-ios-sdk-samples/Layers/Add%20ENC%20exchange%20set#offline-data

However, the URL should link to the download page, not the AGOL portal item. The purpose is to let a developer know where to find the source-of-truth hydro dataset.